packages/cli/src/commands/list.ts

```diff
@@ -230,23 +238,21 @@ export default async function main(client) {
               getProjectName(dep),
               chalk.bold((includeScheme ? 'https://' : '') + dep.url),
               stateString(dep.state),
-              chalk.gray(ms(Date.now() - new Date(dep.createdAt))),
+              chalk.gray(ms(Date.now() - dep.createdAt)),
               dep.creator.username,
             ],
           ])
           // flatten since the previous step returns a nested
           // array of the deployment and (optionally) its instances
-          .reduce((ac, c) => ac.concat(c), [])
-          .filter(
-            app == null
-              ? // if an app wasn't supplied to filter by,
-                // we only want to render one deployment per app
-                filterUniqueApps()
-              : () => true
+          .flat()
```

Comment:
@TooTallNate  @styfle  Here is no necessary use the `reduce` for getting flatten the array. The `flat` method is the **best option** for this scenario also designed to flatten any level nesting of a `array` 😄